src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+const listOfEmployees = document.querySelector('ul');
+let items = document.querySelectorAll('li');
+
+function sortList(list) {
+  listOfEmployees.innerHTML = '';
+
+  const sortedList = [...list].sort((a, b) => {
+    const salaryA = Number(a.dataset.salary.slice(1).split(',').join(''));
+    const salaryB = Number(b.dataset.salary.slice(1).split(',').join(''));
+
+    return salaryB - salaryA;
+  });
+
+  sortedList.forEach((item) => listOfEmployees.append(item));
+  items = document.querySelectorAll('li');
+}
+
+function getEmployees(list) {
+  const sortedArray = [...list];
+
+  return sortedArray;
+}
+
+sortList(items);
+getEmployees(items);
```

Comment:
Same about the getEmployees function. Why do you need it?
```suggestion
sortList(items);
```

Reply:
> Same about the getEmployees function. Why do you need it?

I did it because it is in requirement "
3) Create function `getEmployees`, which takes a `list` property, and returns an array of right objects.
4) Call both functions.
"